src/controllers/applicant.ts

```diff
@@ -0,0 +1,46 @@
+import Applicant, { IApplicant } from '../models/applicant';
+
+export const create_post = async (req, res) => {
```

Comment:
Naming convention should probably be more like createApplicant or something - this is a little confusing because we could eventually import 2 controllers and have some naming restrictions.